Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Android/CrashesInternal.cs

```diff
@@ -106,6 +107,18 @@ public static void StartCrashes()
             AppCenterInternal.Start(AppCenter.Crashes);
         }
 
+        public static ErrorReport BuildHandledErrorReport(string errorReportId)
+        {
+            var nativeErrorReport = _wrapperSdkExceptionManager.CallStatic<AndroidJavaObject>("buildHandledErrorReport", AndroidUtility.GetAndroidContext(), errorReportId);
+            return JavaObjectsConverter.ConvertErrorReport(nativeErrorReport);
```

Comment:
Ok I don't like the isCrash = true optinal parameter for internal code for this reason: here it should have been false and it was missed.

Reply:
Crashes.cs line 391 for all platforms